flow/workflows/cdc_flow.go

```diff
@@ -485,6 +485,8 @@ func CDCFlowWorkflow(
 
 		// if initial_copy_only is opted for, we end the flow here.
 		if cfg.InitialSnapshotOnly {
+			logger.Info("initial snapshot only, ending flow")
+			state.CurrentFlowStatus = protos.FlowStatus_STATUS_COMPLETED
```

Comment:
@Amogh-Bharadwaj does the status ever go to running?

Reply:
Nope

Reply:
or does it go from snapshot to completed directly?

Reply:
it wouldn't, the workflow exits for good here I believe

Reply:
https://github.com/PeerDB-io/peerdb/pull/2275
just to be safe